### PR TITLE
Particle Accelerator

### PR DESCRIPTION
**Pull request by ThatDan123**
image

CL: Added particle accelerator, can be obtained through cargo

-Adds singularity generator to cargo

-Adds radiation collector circuit board to circuit imprinter

-Fixes Geiger counter sounds

Current issues: 

~-PA wont work without voltage, but wires aren't feeding the controller for some reason, bod help pls~

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/Construction/WrenchSecurable.cs [hunk 1]
```diff
@@ -14,15 +14,14 @@ public class WrenchSecurable : NetworkBehaviour, ICheckedInteractable<HandApply>
...
-
 		private HandApply currentInteraction;
 		private string objectName;
 
 		/// <summary>
 		/// Invoked after the anchored state is changed.
 		/// </summary>
-		[NonSerialized]
-		public UnityEvent OnAnchoredChange = new UnityEvent();
+		[NonSerialized] public UnityEvent OnAnchoredChange = new UnityEvent();
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/Construction/WrenchSecurable.cs:23]: why use an UnityEvent if it is not going to be serialized?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Systems/Construction/WrenchSecurable.cs:23]: Thats just a formatting change, i didnt change that. Whoever made it wanted it nonSerialized i guess?

**Comment by corp-0** [comment]
I'd wait until this 
> PA wont work without voltage, but wires aren't feeding the controller for some reason, bod help pls

is fixed to merge. I will add a DNM for now.

**Comment by ThatDan123** [comment]
Good to be merged when ready

**Comment by corp-0** [comment]
Looks good, let''s give it a roll on staging.

File: UnityProject/Assets/Scripts/Objects/Engineering/ParticleAcceleratorControl.cs [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+			ChangePower(ParticleAcceleratorState.Off);
+		}
+
+		private bool TryFindParts()
+		{
+			var enumValues = Enum.GetValues(typeof(OrientationEnum));
+			bool correctArrangement = false;
+
+			//For each set up direction
+			foreach (var enumDirection in enumValues)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Engineering/ParticleAcceleratorControl.cs:167]: this nesting makes me angry tho haha

**Pull request merged**